schemas/glean/glean/glean.1.schema.json

```diff
@@ -3,6 +3,12 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "additionalProperties": false,
   "description": "Schema for the ping content sent by Mozilla's glean telemetry SDK\n",
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "glean",
```

Comment:
Setting `bq_dataset_family` and `bq_table` in the glean schema is a bit confusing because those values are presumably always going to be overridden. It is perhaps a bit confusing already that the `glean` schema here doesn't correspond to any actual tables and might have made more sense to put in `metadata`.

Reply:
/CC @sean-rose 

https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/schema/stable_table_schema.py#L82 is currently relying on the absence of this metadata so we probably need to update our code to skip glean schemas a different way. I still think the `bq_dataset_family` and `bq_table` fields should probably be removed here.

Reply:
I've submitted #805 and mozilla/bigquery-etl#5037 to fix.